Dockerfile

```diff
@@ -1,11 +1,29 @@
 # This is slightly adjusted Dockerfile from
 # https://github.com/docker-library/postgres
 
-# set ALPINE_VERSION= [ edge 3.20 3.19 3.18 3.17 3.16 3.15 3.14 ]
-ARG ALPINE_VERSION=3.20
-FROM alpine:${ALPINE_VERSION}
+# Base image configuration
+# BASE_IMAGE=alpine
+# BASE_VERSION= [ edge 3.20 3.19 3.18 3.17 3.16 3.15 3.14 3.13]
+# New recommended way:
+#   docker build --build-arg BASE_IMAGE=alpine --build-arg BASE_VERSION=3.20 -t orioletest:alpine-3.20 .
+ARG BASE_IMAGE=alpine
+ARG BASE_VERSION=3.20
```

Comment:
It seems that default values of variables are not used. Because of that the following command doesn't work anymore (it is mentioned [in the documentation)](https://github.com/orioledb/orioledb/blob/2cf58ec61ac0cba14212edc14ed567fdce51e0f1/doc/contributing/docker-builds.mdx?plain=1#L24):
```
docker build -t orioletest:17 --pull --network=host --progress=plain --build-arg PG_MAJOR="17" .

#0 building with "orbstack" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 11.76kB done
#1 WARN: InvalidDefaultArgInFrom: Default value for ARG ${BASE_IMAGE}:${BASE_VERSION} results in empty or invalid base image name (line 19)
#1 DONE 0.0s
Dockerfile:19
--------------------
  17 |     ARG BASE_VERSION=${ALPINE_VERSION:+${ALPINE_VERSION}}
  18 |
  19 | >>> FROM ${BASE_IMAGE}:${BASE_VERSION}
  20 |
  21 |     # Re-declare ARGs after FROM for use in subsequent instructions
--------------------
ERROR: failed to solve: failed to parse stage name ":": invalid reference format
```
Do you know if it is possible to use the default version?

Reply:
thank you - fixed!